doc/user-journeys/self-hosted-aws.md

```diff
@@ -0,0 +1,99 @@
+# Deployment Athens Self-Hosted on AWS
+
+The Athens Self-Hosted server can be deployed on AWS using a [CloudFormation](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/Welcome.html) template we provide.
+
+## TODOs
+
+- make multiple instances actually work
+- add stack name or something to snapshot delete, policy
+- consider refactoring to https://github.com/nathanpeck/ecs-cloudformation#publically-networked-service-with-public-load-balancer
+- should using a nginx container as well to filter traffic to athens by http and so that we don't have to set the port on Athens.
+- https cert support & instructions
+
+## Pre-requisites
+
+You will need to [create and activate an AWS account](https://aws.amazon.com/premiumsupport/knowledge-center/create-and-activate-aws-account/).
+Each new account starts with a [free tier](https://aws.amazon.com/premiumsupport/knowledge-center/what-is-free-tier/) that you can use to run Athens.
+You can see how much costs you've incurred in the [Billing page](https://console.aws.amazon.com/billing/).
+
+Next install the [AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/install-cliv2.html) for your operating system, and use the [configure command](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-quickstart.html#cli-configure-quickstart-config) to finish setup.
+The region you choose here is the region that Athens will be installed in.
+
+We'll use `athens-demo` as a name for your Athens deployment, but you can use any other name.
+
+
+## Create
+
+Run the following command to automatically set up an Athens stack.
+
+```sh
+aws cloudformation create-stack \
+    --capabilities CAPABILITY_NAMED_IAM \
+    --template-url https://raw.githubusercontent.com/athensresearch/athens/main/data/self-hosted.cloudformation.yml \
```

Comment:
This URL doesn't exist yet because this PR isn't merged into main.

For now https://raw.githubusercontent.com/filipesilva/athens/aws-deployment/data/self-hosted.cloudformation.yml can be used instead.

If you want to edit this file on disk, you can use `--template-body file://./data/self-hosted.cloudformation.yml` instead.

Reply:
Actually the AWS cli only accepts AWS S3 urls here. I've uploaded one to a bucker of mine.